workflow-templates/npm-audit.yml

```diff
@@ -0,0 +1,18 @@
+name: npm Audit
+
+on:
+  pull_request:
+    paths:
+      - 'package.json'
+      - 'package-lock.json'
+
+jobs:
+  audit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: npm install
+      - name: Run npm audit
+        run: npm audit
+        continue-on-error: false  # Ensure PRs are blocked if vulnerabilities are found
```

Comment:
We can skip this for now.
Sometimes there won't be fixes yet for new vulnerabilities

Reply:
Ok  @drtechie  sir, I have done it, please have a look.